### PR TITLE
287 create in call quality guide for unity

### PR DESCRIPTION
**Pull request by hussain-khalid**

### CODE REVIEW

File: shared/video-sdk/develop/ensure-channel-quality/project-implementation/unity.mdx [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+    To programmatically access the UI elements, add the following at the end of `SetupUI` method:
+
+    ```csharp
+    // Access the video quality button.
+    go = GameObject.Find("videoQuality");
+    // Access the label of the video quality button.
+    videoQuality = go.GetComponentInChildren<TextMeshProUGUI>(true);
+    // Change the video quality button text and font size.
+    videoQuality.text = "Low Video Quality";
+    videoQuality.fontSize = 14;
```
Comment (billy-the-fish) [inline shared/video-sdk/develop/ensure-channel-quality/project-implementation/unity.mdx:83]: Can you remove the font size stuff please.

File: shared/video-sdk/develop/ensure-channel-quality/project-implementation/unity.mdx [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+            Debug.Log(msg);
+    }
+    ```
+
+    Each event reports the statistics of the audio video streams from each remote user and host. 
+
+
+1. **Switch stream quality when the user taps the remote video**
+
+    To take advantage of dual-stream mode and switch remote video quality to high or low, add the following to the `JoinChannelVideo` class:
```
Comment (billy-the-fish) [inline shared/video-sdk/develop/ensure-channel-quality/project-implementation/unity.mdx:232]: Switch video mode and obtain the uid are two different things, can you update as steps please.

**Review by billy-the-fish — CHANGES_REQUESTED** [review]
Good work, looks good, nearly there. The comments are for Video/ILS and voice please. 

Can you pull from main to the milestone branch please so we don't have a massive list of files that have changed please.

File: shared/video-sdk/develop/ensure-channel-quality/project-implementation/unity.mdx (hunk 1 above)
Comment (hussain-khalid) [reply shared/video-sdk/develop/ensure-channel-quality/project-implementation/unity.mdx:83]: Help me here. Can you tell me that where should i move this fontSize stuff?

**Pull request merged**